src/main/java/jdbchomework/console/MenuLauncher.java

```diff
@@ -13,5 +18,14 @@ public MenuLauncher(MainMenu mainMenu, MenuContents menuContents) {
     public void consoleMain() {
         menuContents.makeMenus();
         mainMenu.menu();
+        try {
+            reader.close();
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Use a logger to log this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1148)